--- lineakd-0.7/acinclude.m4.orig	Fri Oct 17 12:55:43 2003
+++ lineakd-0.7/acinclude.m4	Fri Oct 17 12:54:41 2003
@@ -4555,8 +4555,8 @@
        [m4_define([$1], [mm_car($2)])$3[]_mm_foreach([$1],
                                                      mm_car2(m4_shift($2)),
                                                      [$3])])])
-m4_define([AC_FOREACH],
-[mm_foreach([$1], m4_split(m4_normalize([$2])), [$3])])
+dnl m4_define([AC_FOREACH],
+dnl [mm_foreach([$1], m4_split(m4_normalize([$2])), [$3])])
 
 AC_DEFUN(KDE_NEED_FLEX,
 [
