--- lineakd-0.8.4/lineak/Makefile.am.orig	2005-03-02 19:35:46.000000000 +0100
+++ lineakd-0.8.4/lineak/Makefile.am	2006-03-09 23:19:49.543431250 +0100
@@ -8,7 +8,8 @@
 lib_LTLIBRARIES = liblineak.la
   
 liblineak_la_METASOURCES = AUTO
-  
+liblineak_la_LIBADD = -ldl $(X_LIBS) -lXext -lX11 $(X_EXTRA_LIBS)
+
 liblineak_la_SOURCES = pluginmanager.cpp globals.cpp displayctrl.cpp xmgr.cpp cdromctrl.cpp lineak_core_functions.cpp lineak_util_functions.cpp lockctrl.cpp lcommand.cpp saver.cpp msgpasser.cpp loader.cpp lobject.cpp lkey.cpp lbutton.cpp lkbd.cpp ldef.cpp lconfigdata.cpp lconfig.cpp defloader.cpp configloader.cpp configdirectives.cpp commandexec.cpp plugin_definitions.cpp
   
 liblineak_la_LDFLAGS = -version-info 8:0:8 $(all_libraries)
--- lineakd-0.8.4/lineak/xmgr.h.orig	2005-05-25 21:01:24.000000000 +0200
+++ lineakd-0.8.4/lineak/xmgr.h	2006-03-09 23:18:32.366608000 +0100
@@ -35,7 +35,7 @@
 //#include <X11/Xatom.h>
 //#include <X11/Xproto.h>
 #include <X11/XKBlib.h>
-#include <X11/extensions/XKBfile.h>
+//#include <X11/extensions/XKBfile.h>
 }
 
 #include <string>
--- lineakd-0.8.4/lineakd/Makefile.am.orig	2006-03-09 22:18:57.315181000 +0100
+++ lineakd-0.8.4/lineakd/Makefile.am	2006-03-09 23:21:45.602684500 +0100
@@ -2,7 +2,7 @@
 
 lineakd_SOURCES = lineakd_core_functions.cpp cmdprefs.cpp main.cpp 
 lineakd_LDADD   = -lpthread -llineak -lXext -lX11 -ldl
-lineakd_LDFLAGS = -rdynamic -L../lineak $(all_libraries) @X_LIBS@ @X_PRE_LIBS@
+lineakd_LDFLAGS = -rdynamic -L../lineak $(all_libraries) @X_LIBS@
 
 EXTRA_DIST = main.cpp eakprocs.h cmdprefs.cpp cmdprefs.h lineakkb.def lineakd.1.bz2 OLDCODE lineakd_core_functions.h lineakd_core_functions.cpp 
 
@@ -10,17 +10,12 @@
 DEFS = @DEFS@ -DCONFDIR=\"@sysconfdir@\" -DPLUGINDIR=\"@libdir@/lineakd/plugins\" -DBINDIR=\"@bindir@\"
 AM_CPPFLAGS = -I$(top_srcdir) -I$(top_srcdir)/lineak -I$(top_srcdir)/lineakd -I. -I.. $(all_includes) 
 
-#man_MANS = lineakd.1.gz
+man1_MANS = lineakd.1
 
-lineakd.1.bz2:
-	bzip2 -c lineakd.1 > lineakd.1.bz2
-
-install-data-local: lineakd.1.bz2
+install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(sysconfdir)/
-	$(mkinstalldirs) $(DESTDIR)$(mandir)/man1/
 	$(mkinstalldirs) $(DESTDIR)$(pkglibdir)/plugins/
 	$(INSTALL_DATA) $(srcdir)/lineakkb.def $(DESTDIR)$(sysconfdir)/lineakkb.def
-	$(INSTALL_DATA) $(srcdir)/lineakd.1.bz2 $(DESTDIR)$(mandir)/man1/lineakd.1.bz2
 	 if test "$(SETKEYCODES_SUID)" != "no" ; then \
            chown root `which setkeycodes` ; \
            chmod +s `which setkeycodes` ; \
@@ -28,5 +23,4 @@
 
 uninstall-local:
 	-rm -f $(DESTDIR)$(sysconfdir)/lineakkb.def
-	-rm -f $(DESTDIR)$(mandir)/man1/lineakd.1.bz2
 
--- lineakd-0.8.4/utils/Makefile.am.orig	2006-03-09 22:18:57.315181000 +0100
+++ lineakd-0.8.4/utils/Makefile.am	2006-03-09 23:24:02.155218500 +0100
@@ -21,9 +21,9 @@
    send_to_keyboard_SOURCES = send_to_keyboard.c
 endif
 
-xsendkeys_LDADD = -L/usr/X11R6/lib -L$(top_srcdir)/lineak -lXtst -llineak -lX11
-xsendkeycode_LDADD = -L/usr/X11R6/lib -L$(top_srcdir)/lineak -lXtst -lX11
-#lsendcommand_LDADD = -L/usr/X11R6/lib -L$(top_srcdir)/lineak -llineak -lX11
+xsendkeys_LDADD = $(X_LIBS) $(top_builddir)/lineak/liblineak.la -lXtst -llineak -lX11
+xsendkeycode_LDADD = $(X_LIBS) -lXtst -lX11
+#lsendcommand_LDADD = $(X_LIBS) $(top_builddir)/lineak/liblineak.la -lX11
 
 xsendkeys_LDFLAGS = $(AM_CPPFLAGS) 
 xsendkeycode_LDFLAGS = $(AM_CPPFLAGS)
@@ -51,11 +51,11 @@
 
 #install-data-local: send_to_keyboard.8.bz2 evtest.1.bz2 xsendkeycode.8.bz2 xsendkeys.8.bz2
 if EVTEST
-install-data-local: send_to_keyboard.8.bz2 evtest.1.bz2 xsendkeycode.8.bz2 xsendkeys.8.bz2
+install-data-local: send_to_keyboard.8 evtest.1 xsendkeycode.8 xsendkeys.8
 	$(mkinstalldirs) $(DESTIDR)$(mandir)/man1/
-	$(INSTALL_DATA) $(srcdir)/evtest.1.bz2 $(DESTDIR)$(mandir)/man1/evtest.1.bz2
+	$(INSTALL_DATA) $(srcdir)/evtest.1 $(DESTDIR)$(mandir)/man1/evtest.1
 else
-install-data-local: send_to_keyboard.8.bz2 xsendkeycode.8.bz2 xsendkeys.8.bz2
+install-data-local: send_to_keyboard.8 xsendkeycode.8 xsendkeys.8
 endif
 if SEND_TO_KEYBOARD
 	$(mkinstalldirs) $(DESTDIR)$(mandir)/man8/
@@ -64,15 +64,15 @@
 	   chown root $(DESTDIR)$(prefix)/sbin/send_to_keyboard ; \
 	   chmod +s $(DESTDIR)$(prefix)/sbin/send_to_keyboard ; \
 	fi
-	$(INSTALL_DATA) $(srcdir)/send_to_keyboard.8.bz2 $(DESTDIR)$(mandir)/man8/send_to_keyboard.8.bz2
+	$(INSTALL_DATA) $(srcdir)/send_to_keyboard.8 $(DESTDIR)$(mandir)/man8/send_to_keyboard.8
 endif
-	$(INSTALL_DATA) $(srcdir)/xsendkeycode.8.bz2 $(DESTDIR)$(mandir)/man8/xsendkeycode.8.bz2
-	$(INSTALL_DATA) $(srcdir)/xsendkeys.8.bz2 $(DESTDIR)$(mandir)/man8/xsendkeys.8.bz2
+	$(INSTALL_DATA) $(srcdir)/xsendkeycode.8 $(DESTDIR)$(mandir)/man8/xsendkeycode.8
+	$(INSTALL_DATA) $(srcdir)/xsendkeys.8 $(DESTDIR)$(mandir)/man8/xsendkeys.8
 
 uninstall-local:
-	-rm -f $(DESTDIR)$(mandir)/man8/send_to_keyboard.8.bz2
-	-rm -f $(DESTDIR)$(mandir)/man8/xsendkeycode.8.bz2
-	-rm -f $(DESTDIR)$(mandir)/man8/xsendkeys.8.bz2
+	-rm -f $(DESTDIR)$(mandir)/man8/send_to_keyboard.8
+	-rm -f $(DESTDIR)$(mandir)/man8/xsendkeycode.8
+	-rm -f $(DESTDIR)$(mandir)/man8/xsendkeys.8
 if EVTEST
-	-rm -f $(DESTDIR)$(mandir)/man1/evtest.1.bz2
+	-rm -f $(DESTDIR)$(mandir)/man1/evtest.1
 endif
