--- lineakd-0.9/lineak/Makefile.am.orig	2006-05-22 03:45:18.000000000 +0200
+++ lineakd-0.9/lineak/Makefile.am	2006-08-20 21:07:23.124893750 +0200
@@ -19,8 +19,8 @@
 #   liblineak_la_LDFLAGS = -version-info 8:0:8 $(all_libraries); \
 #fi;
 
-liblineak_la_LIBADD  = -lX11 -lXext $(X_LIBS)  -lXt -lSM -lICE -L/usr/X11R6/lib
-liblineak_la_LDFLAGS = -version-info 8:0:8 $(all_libraries) -L/usr/X11R6/lib @X_LIBS@ @X_PRE_LIBS@  -lpthread
+liblineak_la_LIBADD  = $(X_LIBS) -lX11 $(X_EXTRA_LIBS)
+liblineak_la_LDFLAGS = -version-info 8:0:8 $(all_libraries) -lpthread
   
 lineak_HEADERS = configdirectives.h configloader.h definitions.h defloader.h lconfig.h lconfigdata.h ldef.h lobject.h lkbd.h lkey.h lbutton.h loader.h msgpasser.h saver.h lcommand.h lockctrl.h lineak_util_functions.h lineak_core_functions.h cdromctrl.h xmgr.h displayctrl.h plugin_definitions.h commandexec.h pluginmanager.h
   
--- lineakd-0.9/lineakd/Makefile.am.orig	2006-01-30 19:10:43.000000000 +0100
+++ lineakd-0.9/lineakd/Makefile.am	2006-08-20 21:01:36.815250750 +0200
@@ -1,8 +1,8 @@
 bin_PROGRAMS = lineakd
 
 lineakd_SOURCES = lineakd_core_functions.cpp cmdprefs.cpp main.cpp 
-lineakd_LDADD   = -lpthread -llineak -lXext -lX11 -ldl
-lineakd_LDFLAGS = -rdynamic -L../lineak $(all_libraries) @X_LIBS@ @X_PRE_LIBS@
+lineakd_LDADD   = -lpthread $(top_builddir)/lineak/liblineak.la -lX11 -ldl
+lineakd_LDFLAGS = -rdynamic $(all_libraries) @X_LIBS@
 
 EXTRA_DIST = main.cpp eakprocs.h cmdprefs.cpp cmdprefs.h lineakkb.def lineakd.1.bz2 OLDCODE lineakd_core_functions.h lineakd_core_functions.cpp 
 
@@ -10,23 +10,9 @@
 DEFS = @DEFS@ -DCONFDIR=\"@sysconfdir@\" -DPLUGINDIR=\"@libdir@/lineakd/plugins\" -DBINDIR=\"@bindir@\"
 AM_CPPFLAGS = -I$(top_srcdir) -I$(top_srcdir)/lineak -I$(top_srcdir)/lineakd -I. -I.. $(all_includes) 
 
-#man_MANS = lineakd.1.gz
+man_MANS = lineakd.1
 
-lineakd.1.bz2:
-	bzip2 -c lineakd.1 > lineakd.1.bz2
-
-install-data-local: lineakd.1.bz2
-	$(mkinstalldirs) ${DESTDIR}$(sysconfdir)/
-	$(mkinstalldirs) ${DESTDIR}$(mandir)/man1/
-	$(mkinstalldirs) ${DESTDIR}$(pkglibdir)/plugins/
-	$(INSTALL_DATA) $(srcdir)/lineakkb.def ${DESTDIR}$(sysconfdir)/lineakkb.def
-	$(INSTALL_DATA) $(srcdir)/lineakd.1.bz2 ${DESTDIR}$(mandir)/man1/lineakd.1.bz2
-	 if test "$(SETKEYCODES_SUID)" != "no" ; then \
-           chown root `which setkeycodes` ; \
-           chmod +s `which setkeycodes` ; \
-        fi
-
-uninstall-local:
-	-rm -f ${DESTDIR}$(sysconfdir)/lineakkb.def
-	-rm -f ${DESTDIR}$(mandir)/man1/lineakd.1.bz2
+sysconf_DATA = lineakkb.def
 
+install-data-local:
+	$(mkinstalldirs) ${DESTDIR}$(pkglibdir)/plugins
--- lineakd-0.9/utils/Makefile.am.orig	2006-05-22 04:13:48.000000000 +0200
+++ lineakd-0.9/utils/Makefile.am	2006-08-20 21:02:06.617113250 +0200
@@ -1,4 +1,4 @@
-AM_CPPFLAGS = -I$(top_srcdir) -I$(top_srcdir)/lineak -I$(top_srcdir)/lineak/.libs -I$(top_srcdir)/lineakd -I. -I.. -I/usr/include $(all_includes) 
+AM_CPPFLAGS = -I$(top_srcdir) -I$(top_srcdir)/lineak -I$(top_srcdir)/lineakd -I. -I.. -I/usr/include $(all_includes) 
 
 if ! CYGWIN
 
@@ -6,43 +6,18 @@
    bin_PROGRAMS = evtest xsendkeys xsendkeycode
    evtest_SOURCES = evtest.c
    evtest_LDFLAGS = $(AM_CPPFLAGS)
-evtest.1.bz2:
-	bzip2 -c evtest.1 > evtest.1.bz2
-if SEND_TO_KEYBOARD
-install-data-local: send_to_keyboard.8.bz2 evtest.1.bz2 xsendkeycode.8.bz2 xsendkeys.8.bz2
-else
-install-data-local: evtest.1.bz2 xsendkeycode.8.bz2 xsendkeys.8.bz2
-endif
-	$(mkinstalldirs) ${DESTDIR}$(mandir)/man1/
-	$(INSTALL_DATA) $(srcdir)/evtest.1.bz2 ${DESTDIR}$(mandir)/man1/evtest.1.bz2
-	$(mkinstalldirs) ${DESTDIR}$(mandir)/man8/
-	$(INSTALL_DATA) $(srcdir)/xsendkeycode.8.bz2 ${DESTDIR}$(mandir)/man8/xsendkeycode.8.bz2
-	$(INSTALL_DATA) $(srcdir)/xsendkeys.8.bz2 ${DESTDIR}$(mandir)/man8/xsendkeys.8.bz2
+   evtestman = evtest.1
 else
    bin_PROGRAMS = xsendkeys xsendkeycode
-if SEND_TO_KEYBOARD
-install-data-local: send_to_keyboard.8.bz2 xsendkeycode.8.bz2 xsendkeys.8.bz2
-	$(INSTALL_DATA) $(srcdir)/send_to_keyboard.8.bz2 ${DESTDIR}$(mandir)/man8/send_to_keyboard.8.bz2
-else
-install-data-local: xsendkeycode.8.bz2 xsendkeys.8.bz2
-endif
-	$(mkinstalldirs) ${DESTDIR}$(mandir)/man8/
-	$(INSTALL_DATA) $(srcdir)/xsendkeycode.8.bz2 ${DESTDIR}$(mandir)/man8/xsendkeycode.8.bz2
-	$(INSTALL_DATA) $(srcdir)/xsendkeys.8.bz2 ${DESTDIR}$(mandir)/man8/xsendkeys.8.bz2
+   evtestman =
 endif
 
 if SEND_TO_KEYBOARD
 sbin_PROGRAMS = send_to_keyboard
 send_to_keyboard_SOURCES = send_to_keyboard.c
-send_to_keyboard.8.bz2:
-	bzip2 -c send_to_keyboard.8 > send_to_keyboard.8.bz2
-	$(mkinstalldirs) ${DESTDIR}$(mandir)/man8/
-	chmod 755 $(DESTDIR)$(prefix)/sbin/send_to_keyboard
-	if test "$(HELPER_SUID)" != "no" ; then \
-	  chown root $(DESTDIR)$(prefix)/sbin/send_to_keyboard ; \
-	  chmod +s $(DESTDIR)$(prefix)/sbin/send_to_keyboard ; \
-	fi
-
+sendtokeyboardman = send_to_keyboard.8
+else
+sendtokeyboardman = send_to_keyboard.8
 endif
 endif
 
@@ -50,29 +26,16 @@
 xsendkeycode_SOURCES = xsendkeycode.c
 #lsendcommand_SOURCES = lsendcommand.cpp
 
-xsendkeys_LDADD = $(X_LIBS) -L$(top_srcdir)/lineak -lXtst -llineak -lX11
-xsendkeycode_LDADD = $(X_LIBS) -L$(top_srcdir)/lineak -lXtst -lX11
-#lsendcommand_LDADD = $(X_LIBS) -L$(top_srcdir)/lineak -llineak -lX11
+xsendkeys_LDADD = $(X_LIBS) $(top_builddir)/lineak/liblineak.la -lXtst -lX11
+xsendkeycode_LDADD = $(X_LIBS) $(top_builddir)/lineak/liblineak.la -lXtst -lX11
+#lsendcommand_LDADD = $(X_LIBS) $(top_builddir)/lineak/liblineak.la -lX11
 
 xsendkeys_LDFLAGS = $(AM_CPPFLAGS) 
 xsendkeycode_LDFLAGS = $(AM_CPPFLAGS)
 
+man_MANS = xsendkeycode.8 xsendkeys.8 $(evtestman) $(sendtokeyboardman)
 
 # set the include path found by configure
 INCLUDES= $(all_includes) -I$(top_srcdir)/lineak -I. -I..
 DEFS = @DEFS@ -DCONFDIR=\"@sysconfdir@\" -DPLUGINDIR=\"@libdir@/lineakd/plugins\"
 helper_suid = $(HELPER_SUID)
-
-xsendkeycode.8.bz2:
-	bzip2 -c xsendkeycode.8 > xsendkeycode.8.bz2
-xsendkeys.8.bz2:
-	bzip2 -c xsendkeys.8 > xsendkeys.8.bz2
-
-
-uninstall-local:
-	-rm -f ${DESTDIR}$(mandir)/man8/send_to_keyboard.8.bz2
-	-rm -f ${DESTDIR}$(mandir)/man8/xsendkeycode.8.bz2
-	-rm -f ${DESTDIR}$(mandir)/man8/xsendkeys.8.bz2
-if EVTEST
-	-rm -f ${DESTDIR}$(mandir)/man1/evtest.1.bz2
-endif
